### DISCUSSION TITLE
How use bootstrap tooltip in a livewire component?

### DISCUSSION THREAD

**Discussion by jochemgruter** [livewire/livewire#2085] [1. Help] [answered]
I want to use the bootstrap tooltips (as described in the link) below in my livewire component. But after the component is rendered through ajax, the tooltip isn't working anymore. Using `wire:ignore` is not an option because the tooltip is applied on dynamic content.

https://getbootstrap.com/docs/4.5/components/tooltips/

Are there any possibilities to use this?

**Comment by joshhanley** [comment, marked as answer]
It looks like the tooltip appends to the body by default, you probably need to create a container inside your component that has wire:ignore on it, and specify the custom container in the tooltip config.

If that doesn't work, then it is probably because the tooltip handler is getting wiped each time Livewire sends a network request. Wire:ignore is the best bet here, but then you don't have control of the element contents inside Livewire and instead need to update its details using javascript.

You could also try using a manual tooltip trigger instead and use something like AlpineJS to trigger the tooltip manually on hover or click depending on what you need.

Hope this helps!

**Comment by jochemgruter** [comment]
Thanks `wire:ignore` is working for me. I tried it before but I put it on the tooltip itself. Now I put in on the parent element and it is working fine.
